website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -136,14 +136,15 @@ sign-in time based on the user's IdP-provided group membership information.
 
 
 ## FAQs
-- **When are IdP group memberships updated for SSO Mapped groups?**  <br />
-  Group memberships are updated every time a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud for these group memberships to be synchronized.
-
-- **Can I set up SSO without RBAC?**  <br />
-  Yes, see the documentation on [Manual Assignment](#manual-assignment) above for more information on using SSO without RBAC.
-
-- **Can I configure a user's License Type based on IdP Attributes?** <br />
-  Yes, see the docs on [managing license types](/cloud-seats-and-users#managing-license-types) for more information.
-
-- **Why can't I edit a user's group membership?**  <br />
-Make sure you're not trying to edit your own user as this isn't allowed for security reasons. To edit the group membership of your own user, you'll need a different user to make those changes.
+- **When are IdP group memberships updated for SSO Mapped groups?** Group memberships
+  are updated every time a user logs into dbt Cloud via a supported SSO provider. If
```

Comment:
Group memberships are updated whenever a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud to synchronize these group memberships.